modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -414,6 +450,10 @@ The allowed property types for property type constraints are:
 * `LIST<POINT NOT NULL>`
 * Any closed dynamic union of the above types, e.g. `INTEGER | FLOAT | STRING`.
 
+[NOTE]
+A property type constraint cannot be created for `LIST<VECTOR NOT NULL>`.
```

Comment:
```suggestion
A property type constraint cannot be created for `LIST<VECTOR<TYPE>(DIMENSION) NOT NULL>`.
```
maybe? since the base one have them it feels odd not to have them in the list one 🤷